src/ui/hooks/useBiometricsHook.ts

```diff
@@ -10,19 +10,21 @@ import {
 } from "@aparajita/capacitor-biometric-auth/dist/esm/definitions";
 import i18n from "i18next";
 import { PluginListenerHandle } from "@capacitor/core";
+import { useSelector } from "react-redux";
 import { useActivityTimer } from "../components/AppWrapper/hooks/useActivityTimer";
-import { PreferencesKeys, PreferencesStorage } from "../../core/storage";
+import { getBiometryCacheCache } from "../../store/reducers/biometryCache";
 
 const useBiometricAuth = () => {
   const [biometricInfo, setBiometricInfo] = useState<CheckBiometryResult>();
   const [biometricsIsEnabled, setBiometricsIsEnabled] = useState<
     boolean | undefined
   >(undefined);
   const { setPauseTimestamp } = useActivityTimer();
+  const biometryCacheCache = useSelector(getBiometryCacheCache);
 
   useEffect(() => {
     checkBiometry();
-    checkBiometryInPreferences();
+    checkBiometryInCache();
```

Comment:
As we are using the cache(redux) to know if the biometric is enabled, we dont need to use a async method to get the value from redux, so `checkBiometryInCache` is not needed anymore.
We can have something like:

```
const biometryCacheCache = useSelector(getBiometryCacheCache);
const [biometricsIsEnabled, setBiometricsIsEnabled] = useState(biometryCacheCache);
```

We can also remove `biometricsIsEnabled` state from `useBiometricAuth` as we can access to the `BiometryCache` from any component using `useSelector(getBiometryCacheCache)`

Reply:
agreed that we should just remove `biometricsIsEnabled` and `setBiometricsIsEnabled` and replace it with the direct calls to redux - in this case we won't have this state variable at all here - what do you think @bao-sotatek ?

Reply:
@iFergal I agree. I will fix it